CSS/style.css

```diff
@@ -413,12 +421,84 @@ img:hover {
   cursor: pointer;
 }
 
-.close {
-  font-size: 45px;
-  font-weight: 600;
+.nav-style > .close {
+  font-size: 60px;
   color: white;
-  margin-top: -300px;
-  float: right;
-  margin-right: 25px;
   cursor: pointer;
+  margin: 20px;
+}
+
+.pop {
+  display: none;
+  position: fixed;
+  margin: 0 auto;
+  top: 0;
+  bottom: 0;
+  max-height: 100vh;
+  width: 100vw;
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
+  padding: 50px 16px;
+}
+
+.pop-content {
+  width: 100%;
+  margin: 0 auto;
+  height: 100vh;
```

Comment:
Great job making the requested changes 👏🏽

- [ ] [OPTIONAL] the changes made altered could have deisgn to a full viewport pop-up height. I
![Screenshot 2022-07-27 at 11 29 22](https://user-images.githubusercontent.com/31113416/181201411-cab20c9e-3dc6-4c29-9b49-24e5d7def368.png)
 think adjusting the height to default in **desktop view** could help solve this problem.